common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -228,7 +233,7 @@ def student_view(self, context):
                 'type': child.get_icon_class(),
                 'id': child.scope_ids.usage_id.to_deprecated_string(),
                 'bookmarked': is_bookmarked,
-                'path': " > ".join(display_names + [child.display_name or '']),
+                'path': " > ".join(display_names + [markupsafe.escape(child.display_name_with_default)]),
```

Comment:
@robrap We have talked a lot about escaping at the last possible time (usually in the templates). However, here the display names are being concatenated. Do you think there is a better way to handle this escaping?